services/brig/test/integration/API/Team.hs

```diff
@@ -266,8 +307,6 @@ testInvitationCodeExists brig galley = do
     post (brig . path "/register" . contentJson . body (accept email2 invCode)) !!! do
         const 400 === statusCode
         const (Just "invalid-invitation-code") === fmap Error.label . decodeBody
- where
-    invite email = InvitationRequest email (Name "Bob") Nothing
```

Comment:
Can you motivate the `where` to `let` style change, or revert it? (Most of brig/galley tends to use `let` for things of kind `*`, and `where` for things of kind `* -> ...`. That's of course arbitrary, but it would be nice to not change code style too much within the same service)

Reply:
motivation is that it's a `let` in 30 other places in the same module, with almost exactly the same line.  i changed it for consistency.